lib/util/option.js

```diff
@@ -29,29 +31,40 @@ class Option {
         return option;
     return Option.none();
   }
+
+  get value() {
+    throw new Error('Did you mean to call .get()?');
+  }
 }
 
 class Some extends Option {
+  #value;
+
   /* istanbul ignore next */
   constructor(value) {
     super();
-    this.value = value;
+    this.#value = value;
   }
 
-  get() { return this.value; }
+  get() { return this.#value; }
 
-  map(fn) { return Option.of(fn(this.value)); }
-  filter(predicate) { return predicate(this.value) === true ? this : none; } // eslint-disable-line no-use-before-define
+  map(fn) { return Option.of(fn(this.#value)); }
+  filter(predicate) { return predicate(this.#value) === true ? this : none; } // eslint-disable-line no-use-before-define
 
-  orNull() { return this.value; }
-  orElse() { return this.value; }
-  orElseGet() { return this.value; }
-  orThrow() { return this.value; }
+  orNull() { return this.#value; }
+  orElse() { return this.#value; }
+  orElseGet() { return this.#value; }
+  orThrow() { return this.#value; }
 
   isDefined() { return true; }
   isEmpty() { return false; }
 
-  ifDefined(consumer) { consumer(this.value); }
+  ifDefined(consumer) { consumer(this.#value); }
+
+  'fantasy-land/equals'(that) {
```

Comment:
Would you mind adding a comment connecting this to Ramda? I think it'd be useful for someone reading this to immediately connect `fantasy-land` and Ramda. I'm thinking of a comment like "// Add support for Ramda equals()."